scm-info/xml-resolver/scm.yaml

```diff
@@ -0,0 +1,6 @@
+---
+type: "git"
+uri: "https://github.com/xmlresolver/xmlresolver.git"
+tagMapping:
+  - pattern: ([0-9]+\.[0-9]+\.[0-9]+)
```

Comment:
This should automatically match, although this PR has made me realize I don't have the matching rules documented anywhere, I will update the docs with the rules.

Reply:
https://github.com/redhat-appstudio/jvm-build-service/pull/566

Reply:
Removed